community/php7/APKBUILD

```diff
@@ -137,231 +143,244 @@ _build() {
+	ln -sf phpdbg$_suffix "$subpkgdir"/usr/bin/phpdbg
 }
 
-embed() {
-	pkgdesc="PHP Embedded Library"
+libs() {
+	pkgdesc="$pkgdesc (embedded library)"
 	depends="$pkgname-common"
 
 	mkdir -p "$subpkgdir"/usr/lib
-	mv "$pkgdir"/usr/lib/libphp*.so "$subpkgdir"/usr/lib/
+	mv "$pkgdir"/usr/lib/lib$pkgname.so \
+		"$subpkgdir"/usr/lib/ || return 1
 }
 
 litespeed() {
-	pkgdesc="PHP LiteSpeed SAPI"
+	provides="php-litespeed"
+	pkgdesc="$pkgdesc (litespeed)"
 	depends="$pkgname-common"
 
 	mkdir -p "$subpkgdir"/usr/bin
-	mv "$pkgdir"/usr/bin/lsphp "$subpkgdir"/usr/bin/lsphp7
+	mv "$pkgdir"/usr/bin/lsphp \
+		"$subpkgdir"/usr/bin/lsphp$_suffix || return 1
+	ln -sf lsphp$_suffix "$subpkgdir"/usr/bin/lsphp
 }
 
 cgi() {
-	pkgdesc="PHP Common Gateway Interface"
+	provides="php-cgi"
+	pkgdesc="$pkgdesc (common gateway interface)"
 	depends="$pkgname-common"
 
 	mkdir -p "$subpkgdir"/usr/bin
-	mv "$pkgdir"/usr/bin/php-cgi7 "$subpkgdir"/usr/bin/
+	mv "$pkgdir"/usr/bin/php-cgi$_suffix \
+		 "$subpkgdir"/usr/bin/ || return 1
+	ln -sf /usr/bin/php-cgi$_suffix "$subpkgdir"/usr/bin/php-cgi
 }
 
 fpm() {
-	pkgdesc="PHP FastCGI Process Manager"
+	pkgdesc="$pkgdesc (fastcgi process manager)"
 	depends="$pkgname-common"
 
-	mv "$pkgdir"/var "$subpkgdir"/ || return 1
-
-	mkdir -p "$subpkgdir"/etc/$pkgname \
-		"$subpkgdir"/usr \
-		"$subpkgdir"/var/lib \
+	mkdir -p "$subpkgdir"/usr/share \
+		"$subpkgdir"/etc/$pkgname \
 		"$subpkgdir"/var/log/$pkgname
 
-	mv "$pkgdir"/usr/share/$pkgname/fpm "$subpkgdir"/var/lib/$pkgname/ || return 1
 	mv "$pkgdir"/usr/sbin "$subpkgdir"/usr/ || return 1
-	mv "$pkgdir"/etc/$pkgname/php-fpm* "$subpkgdir"/etc/$pkgname/ || return 1
+	mv "$pkgdir"/usr/share/$pkgname \
+		"$subpkgdir"/usr/share/ || return 1
+	mv "$pkgdir"/etc/$pkgname/php-fpm* \
+		"$subpkgdir"/etc/$pkgname/ || return 1
 
-	local file; for file in php-fpm.conf php-fpm.d/www.conf; do
+	local file;
+	for file in php-fpm.conf php-fpm.d/www.conf; do
 		mv "$subpkgdir"/etc/$pkgname/$file.default \
 			"$subpkgdir"/etc/$pkgname/$file || return 1
 	done
 
-	install -D -m755 "$srcdir"/$pkgname-fpm.initd "$subpkgdir"/etc/init.d/php-fpm7 || return 1
-	install -D -m644 "$srcdir"/$pkgname-fpm.logrotate "$subpkgdir"/etc/logrotate.d/php-fpm7
+	install -D -m755 "$srcdir"/$pkgname-fpm.initd \
+		"$subpkgdir"/etc/init.d/php-fpm$_suffix || return 1
+	install -D -m644 "$srcdir"/$pkgname-fpm.logrotate \
+		"$subpkgdir"/etc/logrotate.d/php-fpm$_suffix || return 1
 }
 
 pear() {
-	pkgdesc="PHP Extension and Application Repository"
+	provides="php-pear"
+	pkgdesc="$pkgdesc (extension and application repository)"
 	depends="$pkgname $pkgname-xml"
 
 	mkdir -p "$subpkgdir"/usr/bin \
 		"$subpkgdir"/etc/$pkgname
 
-	local file; for file in phpize7 pecl pear; do
-		mv "$pkgdir"/usr/bin/$file "$subpkgdir"/usr/bin/
-	done
-	mv "$pkgdir"/etc/$pkgname/pear.conf "$subpkgdir"/etc/$pkgname/ || return 1
-	mv "$pkgdir"/usr/share "$subpkgdir"/usr/
+	mv "$pkgdir"/usr/bin/pecl \
+		"$pkgdir"/usr/bin/pear \
```

Comment:
@jirutka 
pecl and pear are the same in all php versions, so it does not matter for now.